src/server.js

```diff
@@ -28,6 +28,9 @@ app.use(cors({
   exposedHeaders: ['Content-Range', 'X-Content-Range'],
 }));
 
+/* implementing cors for http requests during pre - flight phase */
+app.options('*', cors());
```

Comment:
Is it not possible to combine this logic into the already existing `cors` block above this added line? - https://github.com/ijemmao/igbo_api/blob/master/src/server.js#L27-L29

Reply:
Bumping this message. Does it make sense to have two separate instances of cors getting used in this file?

Reply:
I don't know how I'd combine them .. This is how I've been doing it in my project .. Its also same with most code bases I've seen

Reply:
Cool, sounds like it's the norm. Just wanted to double check with you